python/lsst/eo/pipe/biasShiftsTask.py

```diff
@@ -0,0 +1,286 @@
+from collections import defaultdict
```

Comment:
Please delete the unused imports.  Running `flake8` will identify them as well as a number of formatting issues.